content/api/users/users_info.md

```diff
@@ -8,3 +8,23 @@ url = "api-users-info"
   identifier = "api-users-info"
   parent = "api_users"
 +++
+
+Returns an information about the named registered user.
```

Comment:
`Returns information about...`

Reply:
Thank you for reviewing.
I have fixed it.